web-client/src/translations/translations.json

```diff
@@ -13,6 +13,15 @@
         "pleaseWaitLabel": "Please wait...",
         "tip_title": "Remember",
         "tip_info": "Once a volunteer offers to help, you must accept their help in order for them to see your contact information."
+      },
+      "roleinfo": {
+        "title": "Reach4Help",
+        "sub_title": "How can we assist you ?",
+        "info": "Please tell us what is your purpose to use Reach4Help",
+        "info_pin": "I'm here because I need help",
+        "c2a_pin": "Get help now!",
+        "info_cav": "I'm here because I want to help people",
+        "c2a_cav": "Start helping now!"
```

Comment:
This is just a detail, but I usually prefer that the translation key is easily paired with what it translates to. For example

```
// You can abbreviate sometimes
"please_tell_purpose": "Please tell us what is your purpose to use Reach4Help",

// Or use the full text
"im_here_because_i_need_help" : "I'm here because I need help",
```

Let me know what you think about this. It's really just preference 😄

Reply:
I think it's a good concept, I just don't think it's usable if you're translating a complete sentence or so, but I agree the keys can be improved to better pass the meaning of the translation, and I think your idea is quite useful. I would just try to make it short.